src/ts/content/adapters/tags/getTagSheets/formats/impl/zhane.ts

```diff
@@ -0,0 +1,19 @@
+import {FormatStrategy} from "../index";
+import {showToast, ToastType} from "../../../../../../common/ui/toast";
+
+/**
+ * This represents a format that is being used by the Tag Index, but it not
+ * yet recognized by Better LibraryThing.
+ *
+ * This could happen if the extension is out of date.
+ */
+
+const WARNING =
+	"hey! it seems like your installation of Better LibraryThing is a little out of date.\nclick here for update information!";
+
+const zhane: FormatStrategy = async () => {
+	showToast(WARNING, ToastType.WARNING, () => window.open("https://betterlibrarything.com/"));
+	return [];
+};
```

Comment:
are we effectively preventing them from doing anything with the sheet (which is a lot of features, so prevent them from using the extension) if their extension installation is out of date? i know it SHOULD update automatically, but it's a little onerous for a user to figure out how to do it manually or why they're being asked to do it. some people might just ignore it altogether and they'd be in a weird state (right?)

Reply:
I think the problem is, _if_ the format of the tag sheet changes and the extension is out of date, it can't parse and we can't do anything else but let them know why we can't parse the tags.

Ideally, we would ship a new parser _long_ before we actually update the sheet, to ensure everyone is on the latest version, giving us a smooth transition.